src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -135,7 +135,7 @@ private void createEmbeddingsDirectory() {
     private void copyResources(String resourceLocation, String... resources) {
         for (String resource : resources) {
             File tempFile = new File(configuration.getReportDirectory().getAbsoluteFile(),
-                    BASE_DIRECTORY + File.separatorChar + resourceLocation + File.separatorChar + resource);
+                    BASE_DIRECTORY + configuration.getDirectoryQualifier() + File.separatorChar + resourceLocation + File.separatorChar + resource);
```

Comment:
and here

Reply:
No, it's not missing; it's not needed - as this is just a suffix to the current directory name. It can be passed as _abcd for example.